src/command.ts

```diff
@@ -202,6 +202,7 @@ export class Command {
     }
     // allow override of detected non-interactive with --interactive flag
     if (getInheritedOption(options, "interactive")) {
+      options.interactive = true;
```

Comment:
If `interactive` is change to be a flag of the `ext:install` command, I don't think this line is necessary.

Reply:
I went back to API council and they approved adding `-i` globally.